examples/oauth/certs/ca.crl

```diff
@@ -0,0 +1,16 @@
+-----BEGIN X509 CRL-----
```

Comment:
This file and some other new files are not used.

Also, you could reuse the test TLS files in [`testdata`](https://github.com/grpc/grpc-go/tree/master/testdata). Sample of usage: [client](https://github.com/grpc/grpc-go/blob/6e5192d28c2b6a60ea6c2d6fbf1e128b0a65ee3d/interop/client/client.go#L85), [server](https://github.com/grpc/grpc-go/blob/6e5192d28c2b6a60ea6c2d6fbf1e128b0a65ee3d/interop/server/server.go#L56).

Reply:
Yes! I didn't realize there were certs in the project. I'll use those.